### ISSUE TITLE
Crash with Gnome calendar 3.32.0

### ISSUE THREAD

**Issue by sdeleuze** [etesync/etesync-dav#69]
When I try to connect Gnome Calendar 3.32.0 to etesync-dav 0.5.0, the application just crash. That seems to be a pretty usual use case, is there some known incompatibilities?

**Comment by tasn** [comment]
Which of the applications crash? Gnome Calendar or etesync-dav? Maybe what you're experiencing is related to #66? Though neither of the applications should crash even with that.

**Comment by sdeleuze** [comment]
Gnome calendar

**Comment by tasn** [comment]
Weird, because have used Gnome Calendar before (and Evolution, same backend). I wonder if there's an issue with Gnome Calendar, or maybe it's reacting poorly to #66. Either way, it shouldn't be crashing, you should probably open a ticket with them as well. :|

**Comment by reyman** [comment]
Same here, etesync-dav 0.11.0 work well on localhost, but Gnome Calendar 3.32.2 crash when try to configure on Ubuntu 18.04.

**Comment by tasn** [comment]
3.32.2 works here just fine. I just tried.
I also have:
evolution 3.32.4
evolution-data-server 3.32.4

What about you? Are you able to come to IRC for some debugging and trying to figure this out?

**Comment by tasn** [comment]
Also, I just checked, Ubuntu 18.04 doesn't have 3.32.2. Have you used backports? If I had to guess: it's a bug in gnome/evolution-data-server or one other of its components that has now been fixed.

**Comment by reyman** [comment]
Sorry i have to edit, it's 19.04, i updated my version of Ubuntu today

**Comment by reyman** [comment]
Finally it's work, you need to add your calendar with evolution and DAV option. After that your calendar appear in gnome-calendar. Gnome-calendar don't propose such dav option, and the web option don't work.

**Comment by tasn** [comment]
OK, after a quick investigation on IRC: it's a combination of user error and buggy gnome-calendar.
First of all: gnome-calendar and contacts do *not* support adding cal/carddav calendars. In order to add them you need to add them from Evolution (or maybe gnome-settings too?). "From the web" is different to "DAV". So EteSync works and always has. I'll clarify the docs now.

As for gnome-calendar crashing: doesn't seem to be crashing here even when I use "from the web", maybe they fixed it. I have no idea, but either way, that's not the correct setting.

TL;DR: Configure gnome-calendar/contacts from evolution. I'll now update the README.
